static/i18n/en/app.json

```
@@ -283,9 +283,10 @@
         "cta": "Continue",
         "pending": {
           "cta": "Continue",
-          "title": "Pending approval",
-          "subtitle": "Your information was submitted for approval by Wyre.",
-          "link": "Learn more about KYC"
+          "title": "Your information has been submitted for approval",
+          "subtitle": "This usually takes 2-3 days. You will receive an email from Wyre once your identity is verified succesfully.",
```

"subtitle": "This usually takes 2-3 days. You will receive an email from Wyre once your identity is verified succesfully."

---

Yes, this lingering 2-3 days wording has been wrong from the beginning
```suggestion
          "subtitle": "It usually takes a few minutes for the review to be completed.",
```